.travis.yml

```diff
@@ -1,5 +1,7 @@
 sudo: false
 language: python
+python:
+    - 'nightly'
```

Comment:
I tried reverting this change, and Tox said 'ERROR: InterpreterNotFound: python3.5'. So I put it back.